apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```diff
@@ -1461,12 +1442,6 @@ class MapEdit extends MapView {
                     firstSectionUnit.index = '1'
```

Comment:
Why is this part left? In units you removed it
       
        if (typeof this.map.parking[removedIndex] !== 'undefined') {
            if (removedIndex === 0) {
                // Rename from first unit
                const firstSectionUnit = get(last(this.map.parking[removedIndex].floors.filter(floor => floor.index > 0)), 'units.0')
                if (firstSectionUnit) {
                    firstSectionUnit.label = '1'
                    firstSectionUnit.index = '1'
                    this.updateParkingUnitNumbers(firstSectionUnit)
                }